content/community/governance/gcb/index.md

```diff
@@ -0,0 +1,102 @@
+
+Active contributing communities are encouraged to form and govern their own SIG, with a focus on specific shared interests. Each SIG may elect one member to represent their interests on the GCB. This is analogous to the Technical Board, consisting of representatives of the Working Groups (WG).
+
+### Characteristics of SIGs:
+
+* Primarily focused on community development: as distinct from the more technical issues managed by WGs.
+* Flexible model: with simple criteria such as evidence of genuine community support, hub updates and self-regulation,
+for ongoing recognition, easy to create and to dissolve, can be transient
+* Some supporting administrative infrastructure will be provided by the Galaxy Project 
+  * mailing list
+  * website infrastructure and a place on the Galaxy hub to present the Community
+  * presentation slot at the annual GCC conference
+  * Chat channel/room
+* Govern their own activities such as organising meetings, developing training material and tools, maintaining the Hub pages, and choosing a representative for the GCB.
+* Internally organise themselves to gather and articulate their own needs and requirements as part of the Galaxy Project roadmap and (competitive grant) processes.
+
+### Requirements for SIGs
+
+* A SIG must have 2 or more members to be viable.
+* SIGs must maintain a page on the Hub with at least:
+    * A description of the focus and purposes of the SIG.
+    * A clear explanation of how individuals can join and maintain membership.
+    * Details of access to the SIG’s communication channels
+    * A description of roadmap related processes for gathering input from members
+* SIGs are required to provide the current contact/email address for the nominated GCB representative, and to manage membership and internal communication for itself. 
+* SIGs are expected to use at least one public Matrix channel under the Galaxy Matrix space.
+
+### Creating a new SIG
+
+We encourage everyone with interest in Galaxy to join a SIG, a WG or create their own SIG. The Galaxy Project Management Office (PMO)
+is the first point of contact for anyone wanting to form a new SIG. To illustrate the scope of SIGs see the community page on the Galaxy hub and the following examples:
+
+* Microbial (People interested in Microbial research, tool, workflow and training development)
+* Industry Forum (a forum for people to exchange needs and experiences with Galaxy in Industry settings)
+* Indian Galaxy Community (regional Galaxy community, interested in Galaxy community building)
+* AnVIL (funded project related to Galaxy)
+* BLA (Better License Annotation SIG - does not exist yet, but could bring people together to better annotate Galaxy tools and workflows with appropriate licenses, could develop License recommendations)
+
+## Why make a SIG?
```

Comment:
there are good points in this section but the narrative needs to be a more clear I think

Reply:
I think this should be deleted now that there's a nice directories page, just link to that instead to shorten this section